<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>1451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Day temp</string>
    </property>
    <widget class="QSlider" name="dayTempMinSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>40</number>
     </property>
    </widget>
    <widget class="QSlider" name="dayTempMaxSlider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>40</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="daytempMinLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="daytempMaxLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>91</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Night temp</string>
    </property>
    <widget class="QSlider" name="nightTempMinSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>40</number>
     </property>
    </widget>
    <widget class="QSlider" name="nightTempMaxSlider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>40</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="nighttempMaxLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="nighttempMinLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>91</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Feels like temp</string>
    </property>
    <widget class="QSlider" name="feelslikeTempMinSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>40</number>
     </property>
    </widget>
    <widget class="QSlider" name="feelslikeTempMaxSlider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>40</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="feelslikeMaxLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="feelslikeMinLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>91</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Precipitation</string>
    </property>
    <widget class="QSlider" name="precipitationamountValueSlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="precipitationLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>91</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Wind speed</string>
    </property>
    <widget class="QSlider" name="windspeedMinSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSlider" name="windspeedMaxSlider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="windspeedMinLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="windspeedMaxLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>91</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Wind gust speed</string>
    </property>
    <widget class="QSlider" name="windGustSpeedSlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="windgustLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>91</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Pressure</string>
    </property>
    <widget class="QSlider" name="pressureSlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>231</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="pressureLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="windDirComboBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Северный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Северо-восточный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Северо-западный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Южный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Юго-восточный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Юго-западный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Восточный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Западный</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="requiredFlagsGroupBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Flags</string>
    </property>
    <widget class="QCheckBox" name="rainCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Дождь</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="snowCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Снег</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="hailCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Град</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="frostCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Иней</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="icyroadsCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Гололед</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="fogCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Туман</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="stormCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ураган</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="lightningCheckbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Гроза</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="snowstormCheckbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Метель</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="freezingCheckbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Мороз</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dustCheckbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Пыль</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="smogCheckbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Смог</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="icingCheckbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Обледенение</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="emergencyCheckbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ЧС</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="appendRecordButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Append record</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
